app/models/map.rb

```diff
@@ -87,8 +87,8 @@ def latest_export
   def self.authors(limit = 50)
     Map.limit(limit)
        .order("maps.id DESC")
-       .where('password = "" AND archived = "false"')
-       .collect(&:author)
+       .where('password = "" AND archived = "false" AND author!="anonymous"')
+       .collect(&:author).uniq
```

Comment:
this should be good, although i think in ActiveRecord we could also do this, maybe?

```suggestion
       .distinct.collect(&:author)
```

Reply:
oh wait. that's not right. Um, instead -- maybe this? 
```suggestion
       .group(:author).collect(&:author)
```

Reply:
How about
.group(:author).pluck(:author)
?

Reply:
Yes, nice! Let's do that! Sorry i missed your comment originally!

Reply:
```suggestion
       .group(:author).pluck(:author)
```

Reply:
Try that!